Ecosystem Partners/Marketplace Guides/getting-started-with-kemp-technology-loadmaster-template.md

```diff
@@ -0,0 +1,78 @@
+
+### **Key Features**
+* Supported on VMware, Hyper-V, KVM, Xen, KVM and Oracle VirtualBox
+* Server Load Balancing for TCP/UDP based protocols
+* NAT-based forwarding
+* Support for Direct Server Return (DSR) configurations
+* Active Directory integration
+* Configurable S-NAT support
+* Easy administration and configuration<br><br>
+### **Audience**
+CenturyLink Cloud Users, Applications network Administrators, IT Managers<br><br>
+### **Prerequisites**
+* Access to the CenturyLink Cloud platform as an authorized user
+* Control Portal account with password authentication<br>
+
+### **Steps to Deploy a New Virtual LoadMaster**<br>
+----
+TO: Supports@ctl.io
+
+EMAIL SUBJECT:   Custom Image Import Request for Ecosystem Partner Template
+
+CLC Support Team,
+Please create a ticket to complete the following Service Task:
+
+Please import the Ecosystem Partner Template image file referenced below to my CenturyLink Cloud Account:
+- In the ALIAS
+- My CenturyLink Cloud Account Alias: ####
+-- My CenturyLink Cloud Account PIN: ####
+- Data Center to import image to: ###
+- Server Name to import image as: ##########
+- VLAN in the account to add the Server to: ########
+- Number of VIP's you'd like reserved in the network for load balancing.  (The support team can reserve more later via a ticket.  CLC Support team will reserve 10 VIPs unless stated otherwise.)
+
+
+7. Connect to the VLM via SSH for initial licensing an setup.
+8. Change password (default username is "bal" and initial password is blank.
+9. Connect to Virtual LoadMaster™ via Web User Interface by using the appropriate assigned IP address and continue configuration of your application environment.
+
+*For more information on how to configure Virtual LoadMaster™, please visit: [KEMP Support Site](https://www.kemptechnologies.com/support)*<br>
+###**Pricing**
+You may secure entitlements to the technology by contacting [KEMP](mailto:sales@kemptechnologies.com)<br>
+### **Frequently Asked Questions**<br>
+**Where do I obtain a KEMP license?**<br>
+Existing CenturyLink Enterprise Customers can contact their Account Representative for help obtaining a KEMP Technologies license, or contact KEMP Technologies directly:
+* Contact KEMP Technologies Sales via [Email KEMP Sales](mailto:sales@kemptechnologies.com)
+* Contact KEMP Technologies Sales via Phone: 1-844-470-5367<br>
+
+**Who should I contact for support?**
+* For issues related to accessing the KEMP Technologies Virtual LoadMaster™ setup, configuration or for any licensing concerns, please visit the [KEMP Technologies Support Website](http://www.kemptechnologies.com/support)
+* For issues related to cloud infrastructure, please open a ticket using the [CenturyLink Cloud Support Process](https://../../Support/how-do-i-report-a-support-issue.md).<br>

```

Comment:
change the link to `../../Support/how-do-i-report-a-support-issue.md` (no `https`, it should be the filename of the referenced KB article)